README.md

```diff
@@ -47,6 +47,12 @@ Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the
 <p align="center"><img src="/assets/Issue-Done.png"></p>
 
 
+### 👩‍💻💕About Us
+
+Angie and Arlene are LA natives that met while attending Dev Bootcamp in San Francisco.  After bootcamp was over and they were back in LA, they wanted to once again be part of an amazing, welcoming community like DBC was. They found Hoodie through Rails Girls! This project is extra special for them as it is their first contribution to open source.
```

Comment:
maybe instead of Rails Girls, say Rails Girls Summer of Code and link to their website? https://railsgirlssummerofcode.org/

Reply:
@agonzalez0515 Would it be too much if we add our pictures like this? 
![image](https://user-images.githubusercontent.com/8839310/30255881-79054ed0-965b-11e7-9303-fb7876eb836c.png)

Reply:
I can add that once I merge the PR.  Do you mind adding @gr2m 's suggestion so I can merge.